internal/cmd/internal/olmv1/printing.go

```diff
@@ -59,6 +67,9 @@ func printFormattedCatalogs(catalogs ...olmv1.ClusterCatalog) {
 // name (asc), version (desc)
 func sortOperators(extensions []olmv1.ClusterExtension) {
 	slices.SortFunc(extensions, func(a, b olmv1.ClusterExtension) int {
+		if a.Status.Install == nil || b.Status.Install == nil {
+			return cmp.Compare(a.Name, b.Name)
+		}
```

Comment:
This is not related to the `empty client in action config` issue. Can you please add some context in the commit message? The commit message will help others understand the issue when they look back at the issue using `git blame`.